src/fides/data/sample_project/docker-compose.yml

```diff
@@ -24,15 +24,35 @@ services:
         target: /fides/fides_uploads
         read_only: False
 
+  sample-app:
+    build:
+      context: ./cookie_house
+      dockerfile: Dockerfile
+    environment:
+      - PORT=3000
+      - DATABASE_HOST=postgres-test
+      - DATABASE_PORT=5432
+      - DATABASE_USER=postgres
+      - DATABASE_PASSWORD=postgres
+      - DATABASE_DB=postgres_example
+    ports:
+      - 3000:3000
+    depends_on:
+      - postgres-test
+
   fides-pc:
     image: ethyca/fides-privacy-center:sample
     ports:
-      - "3000:3000"
+      - "3001:3000"
```

Comment:
What's the rationale for moving the Privacy Centre to 3001 over sample-app?

Reply:
This'll make it match `test_env`, which I think is good for Ethyca employees' muscle memory